From b4312e9075b8425dbddacc587ec6f4f030a3637c Mon Sep 17 00:00:00 2001
From: Andre Ribeiro <andre.ribeiro.srs@gmail.com>
Date: Sat, 16 Sep 2023 13:24:23 -0300
Subject: [PATCH] On branch lf-5.15.y Changes to be committed: 	modified:  
 arch/arm64/boot/dts/freescale/imx8mn-evk.dtsi 	modified:  
 drivers/dma/fsl-dpaa2-qdma/dpaa2-qdma.c 	modified:  
 drivers/firmware/imx/scu-pd.c 	modified:   include/uapi/linux/hx280enc.h 
 modified:   sound/soc/fsl/fsl_easrc_m2m.c

---
 arch/arm64/boot/dts/freescale/imx8mn-evk.dtsi | 10 ++++++++--
 drivers/dma/fsl-dpaa2-qdma/dpaa2-qdma.c       |  0
 drivers/firmware/imx/scu-pd.c                 |  0
 include/uapi/linux/hx280enc.h                 |  0
 sound/soc/fsl/fsl_easrc_m2m.c                 |  0
 5 files changed, 8 insertions(+), 2 deletions(-)
 mode change 100755 => 100644 drivers/dma/fsl-dpaa2-qdma/dpaa2-qdma.c
 mode change 100755 => 100644 drivers/firmware/imx/scu-pd.c
 mode change 100755 => 100644 include/uapi/linux/hx280enc.h
 mode change 100755 => 100644 sound/soc/fsl/fsl_easrc_m2m.c

diff --git a/arch/arm64/boot/dts/freescale/imx8mn-evk.dtsi b/arch/arm64/boot/dts/freescale/imx8mn-evk.dtsi
index 6c2d66cdd2f4..774476688e6b 100644
--- a/arch/arm64/boot/dts/freescale/imx8mn-evk.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mn-evk.dtsi
@@ -19,7 +19,7 @@ gpio-leds {
 		status {
 			label = "yellow:status";
 			gpios = <&gpio3 16 GPIO_ACTIVE_HIGH>;
-			default-state = "on";
+			linux,default-trigger = "heartbeat";
 		};
 	};
 
@@ -290,6 +290,12 @@ &i2c3 {
 	sda-gpios = <&gpio5 19 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 
+	sensor@48 {
+		compatible = "national,lm75a";
+		reg = <0x48>;
+		status = "okay";
+  	};
+
 	pca6416: gpio@20 {
 		compatible = "ti,tca6416";
 		reg = <0x20>;
@@ -351,7 +357,7 @@ ov5640_mipi_0: ov5640_mipi@3c {
 		mclk = <24000000>;
 		mclk_source = <0>;
 		mipi_csi;
-		status = "okay";
+		status = "disabled";
 		port {
 			ov5640_ep: endpoint {
 				remote-endpoint = <&mipi1_sensor_ep>;
diff --git a/drivers/dma/fsl-dpaa2-qdma/dpaa2-qdma.c b/drivers/dma/fsl-dpaa2-qdma/dpaa2-qdma.c
old mode 100755
new mode 100644
diff --git a/drivers/firmware/imx/scu-pd.c b/drivers/firmware/imx/scu-pd.c
old mode 100755
new mode 100644
diff --git a/include/uapi/linux/hx280enc.h b/include/uapi/linux/hx280enc.h
old mode 100755
new mode 100644
diff --git a/sound/soc/fsl/fsl_easrc_m2m.c b/sound/soc/fsl/fsl_easrc_m2m.c
old mode 100755
new mode 100644
